apps/web-mzima-client/src/app/feed/feed.component.ts

```diff
@@ -314,6 +315,20 @@ export class FeedComponent extends MainViewComponent implements OnInit {
     });
   }
 
+  public postIsLoading(value: boolean) {
+    return (this.isLoading = value); // "There are no posts yet!" flicker is fixed for (most) places where the value here is set to true
```

Comment:
I think you can just skip this function and use this.isLoading = value where needed directly instead.

Reply:
Yes, its a matter of taste really :D It still works, my thinking was that we use this.isLoading in many places already so its good to use the same pattern.

Reply:
All right - resolved.